it-it/typescript-it.html.markdown

```diff
@@ -0,0 +1,226 @@
+---
+language: TypeScript
+contributors:
+    - ["Philippe Vlérick", "https://github.com/pvlerick"]
+translators:
+    - ["Christian Grasso", "https://grasso.io"]
+filename: learntypescript-it.ts
+---
```

Comment:
You need to add the language spec line here for the site builder to pick it up as a translation of an existing document. In its present state, it just replaces the English TypeScript document.

```
lang: it-it
```